debian/control

```diff
@@ -7,5 +7,5 @@ Standards-Version: 3.9.6.1
 
 Package: gpgmailer
 Architecture: all
-Depends: adduser (>= 3.11), confighelper (>=0.8), python-gnupg, python-daemon
+Depends: adduser (>= 3.11),parkbench-common (>=0.8),python-gnupg,python-daemon,postfix | mail-transfer-agent
```

Comment:
Again, list please dependencies in alphabetical order.

Also, how does the bar operator work? Please let me know next time we meet.

Reply:
The [documentation](https://www.debian.org/doc/debian-policy/ch-relationships.html) isn't super clear about this behavior, but the | is an inclusive or. If mail-transfer-agent is provided by some package, the dependency will be satisfied, if it isn't, then it will install Postfix.